src/sec_edgar_bulker/core/header_generator.py

```
@@ -0,0 +1,95 @@
+            Dict[str, str]: Dictionary of HTTP headers.
+            
+        Raises:
+            ValueError: If email domain format is invalid.
+        """
+        if not self.settings.use_generator:
+            return self._static_headers.copy()
+            
+        headers = self._static_headers.copy()
+        
+        if self.settings.random_settings:
+            domain = random.choice(self.settings.random_settings.email_domains)
```

Ruff (0.8.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)